src/overlays/actors/ovl_Boss_Tw/z_boss_tw.c

```diff
@@ -156,183 +187,5337 @@ static ColliderCylinderInit D_8094A840 = {
+                globalCtx->envCtx.unk_D8 = 0.0f;
+                TitleCard_InitBossName(globalCtx, &globalCtx->actorCtx.titleCtx, SEGMENTED_TO_VIRTUAL(D_0602E170), 0xA0,
+                                       0xB4, 0x80, 0x28);
+                gSaveContext.eventChkInf[7] |= 0x20;
+                Audio_QueueSeqCmd(0x1B);
+            }
+
+            if (this->work[CS_TIMER_1] >= 160) {
+                if (this->work[CS_TIMER_1] == 160) {
+                    this->subCamEyeStep.x = 0.0f;
+                }
+                Math_ApproachF(&this->subCamEye.x, 0.0f, 0.05f, this->subCamEyeStep.x * 0.5f);
+                Math_ApproachF(&this->subCamEye.z, 1000.0f, 0.05f, this->subCamEyeStep.x);
+                Math_ApproachF(&this->subCamEyeStep.x, 40.0f, 1.0f, 1);
+            } else {
+                Math_ApproachF(&this->subCamEye.x, 300.0f, 0.05f, this->subCamEyeStep.x);
+                Math_ApproachF(&this->subCamEyeStep.x, 5.0f, 1.0f, 0.5f);
+            }
+
+            if (this->work[CS_TIMER_1] < 200) {
+                Audio_PlayActorSound2(&sKoumePtr->actor, NA_SE_EN_TWINROBA_FLY - SFX_FLAG);
+                Audio_PlayActorSound2(&sKotakePtr->actor, NA_SE_EN_TWINROBA_FLY - SFX_FLAG);
+                sp90.x = this->workf[UNK_F11];
+                sp90.y = 400.0f;
+                sp90.z = 0.0f;
+                Matrix_RotateY(this->workf[UNK_F9], MTXMODE_NEW);
+                Matrix_MultVec3f(&sp90, &sp84);
+                sKoumePtr->actor.world.pos.x = sp84.x;
+                sKoumePtr->actor.world.pos.y = sp84.y;
+                sKoumePtr->actor.world.pos.z = sp84.z;
+                sKoumePtr->actor.world.rot.y = sKoumePtr->actor.shape.rot.y = (this->workf[UNK_F9] / M_PI) * 32768.0f;
+                sKotakePtr->actor.world.pos.x = -sp84.x;
+                sKotakePtr->actor.world.pos.y = sp84.y;
+                sKotakePtr->actor.world.pos.z = -sp84.z;
+                sKotakePtr->actor.shape.rot.y = sKotakePtr->actor.world.rot.y =
+                    ((this->workf[UNK_F9] / M_PI) * 32768.0f) + 32768.0f;
+                Math_ApproachF(&this->workf[UNK_F11], 80.0f, 0.1f, 5.0f);
+                this->workf[UNK_F9] -= this->workf[UNK_F10];
+                Math_ApproachF(&this->workf[UNK_F10], 0.19999999f, 1.0f, 0.0019999994f);
+            }
+
+            if (this->work[CS_TIMER_1] == 200) {
+                sKoumePtr->actionFunc = BossTw_FlyTo;
+                sKotakePtr->actionFunc = BossTw_FlyTo;
+                sKoumePtr->targetPos.x = 600.0f;
+                sKoumePtr->targetPos.y = 400.0f;
+                sKoumePtr->targetPos.z = 0.0f;
+                sKoumePtr->timers[0] = 100;
+                sKotakePtr->targetPos.x = -600.0f;
+                sKotakePtr->targetPos.y = 400.0f;
+                sKotakePtr->targetPos.z = 0.0f;
+                sKotakePtr->timers[0] = 100;
+            }
+
+            if (this->work[CS_TIMER_1] == 260) {
+                Camera* cam = Gameplay_GetCamera(globalCtx, 0);
```

Comment:
```suggestion
                Camera* cam = Gameplay_GetCamera(globalCtx, 0);
                
```

Reply:
think this is also needs MAIN_CAM for 0, just commenting as a note to myself